packages/platform-sdk-profiles/src/drivers/memory/repositories/profile-repository.ts

```diff
@@ -148,4 +149,11 @@ export class ProfileRepository implements IProfileRepository {
 
 		return result;
 	}
+
+	/** {@inheritDoc IProfileRepository.completeTutorial} */
+	public completeTutorial = (profile: IProfile): void => {
+		this.#data.set(ProfileData.HasCompletedTutorial, true);
```

Comment:
```suggestion
		this.#data.set(ProfileData.HasCompletedIntroductoryTutorial, true);
```

Reply:
@faustbrian change enum?

Reply:
Yes, to avoid collisions in case we introduce other tutorials which is likely because we support many different coins.